client/app/scripts/ng-sir-trevor-blocks.js

```diff
@@ -16,18 +16,13 @@ define([
     'use strict';
 
     function isURI(string) {
-        var url_regex = /^(?:([A-Za-z]+):)?(\/{0,3})([0-9.\-A-Za-z]+)(?::(\d+))?(?:\/([^?#]*))?(?:\?([^#]*))?(?:#(.*))?$/;
-        return (url_regex.test(string));
-    }
-    function embedFacebookToUrl(string) {
-        // regular expression to catch a facebook url in the embed code string.
-        // catch protocol even if relative, subdomain, facebook.com hardcoded and the path.
-        var facebook_regex = /(\b(?:([A-Za-z]+):)?\/\/([0-9.\-A-Za-z]+)facebook.com[-A-Za-z0-9+&@#\/%?=~_|!:,.;]*[-A-Za-z0-9+&@#\/%=~_|])/,
-            matches = string.match(facebook_regex);
-        if (matches) {
-            return matches[0];
-        }
-        return string;
+        var pattern = new RegExp('^(https?:\\/\\/)?'+ // protocol
+        '((([a-z\\d]([a-z\\d-]*[a-z\\d])*)\\.?)+[a-z]{2,}|'+ // domain name
```

Comment:
what that did was to get only the href from the fb embed code and pass it on to embedly and that is exactly what we wanted to avoid. it was made last year a fix for something that does not happen anymore from what I saw( doubling the fb embed LBSD-646)